localstack/utils/testutil.py

```diff
@@ -197,7 +197,8 @@ def create_lambda_function(
     region_name=None,
     **kwargs,
 ):
-    """Utility method to create a new function via the Lambda API"""
+    """Utility method to create a new function via the Lambda API
+    CAVEAT: Does NOT wait until the function is ready/active. The fixture create_lambda_function waits until ready."""
```

Comment:
This method is actually the base of create_lambda_function, as the fixture was meant as a drop in replacement. Nobody should use this function directly anymore (or almost anything in testutil, for that matter)

Reply:
Makes sense.
I just saw it used in other services (17x in localstack and 21x in -ext) and therefore added the clarification and reference to the fixture.